spotify_project/base/templates/single_artist.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
By now, you've noticed that a bunch of what is in here is a direct copy/paste from what was in your other template. Time to look into Template Extending. 

By the end of this PR, you'll want to have three total templates.

1. base.html - This will do all the boiler-plate stuff like declaring the doctype, etc. It will also have `block`s to define page_title and body. So those can be overwritten later
2. The template for listing all artists (now updated to extend from base.html
3. This template, also extended from base.html

If you run into issues with getting this working, we can talk about it later when you are here.